extension/src/popups/MinervaMenu.html

```diff
@@ -16,72 +16,102 @@
   <body>
     <div id="extensionBody">
       <div class="collapse" id="minervaMenuCollapseCard">
-        <div class="card card-body">
-          <ul class="nav nav-pills justify-content-center">
-            <li class="nav-item">
-              <a class="nav-link active" id="you-tab" data-toggle="tab" href="#nav-you" role="tab" aria-controls="you"
-                aria-selected="true">You</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" id="ranking-tab" data-toggle="tab" href="#nav-ranking" role="tab" aria-controls="ranking"
-                aria-selected="false">Ranking</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" id="store-tab" data-toggle="tab" href="#nav-store" role="tab" aria-controls="store"
-                aria-selected="false">Store</a>
-            </li>
-          </ul>
-          <div class="tab-content" id="nav-tabContent">
-            <div class="tab-pane fade show active overflow-auto" id="nav-you" role="tabpanel" aria-labelledby="nav-you-tab">
-              <div class="card">
-                <div class="card-body">
-                  <h5 class="card-title" id="studentName-youPanel" ></h5>
-                  <p class="card-text">Total minutes watched: 69:42</p>
+        <div class="container">
+          <div class="row">
+            <div class="col">
+              <div class="card-body" id="side-bar-owned-badges"></div>
```

Comment:
Can we move this into the `You` Tab? This seems irrelevant for the ranking and the store tab

Reply:
I wanted the stickers to be displayed on all pages because that seems most appropriate.
It's not relevant for ranking and store functionality but I think it's relevant for student satisfaction.

Reply:
In such case, could you more the UI more obvious? Currently, the stickers just get lined up on the top in an awkward way. It would be more intuitive if we add a text label saying things like `Owned Badges`. Also, on the same line as your argument, I believe we also need to render coin balance for student satisfaction along with `Owned Badges` since that is relevant across tabs